tox.ini

```diff
@@ -9,8 +9,11 @@ envlist =
 commands =
     zope-testrunner --test-path=src []
     sphinx-build -b doctest -d {envdir}/.cache/doctrees docs {envdir}/.cache/doctest
+extras =
+    test
+    docs
 deps =
-    .[test,docs]
+    py27,py27-watch,pypy: sphinxcontrib-websupport < 1.2
```

Comment:
It's not clear to me why this is here. I don't see us using that extension anywhere?

Reply:
It is a direct dependency of Sphinx (see https://github.com/sphinx-doc/sphinx/blob/591bdd74c0739269a9b2651aec2fa2554a3d8d23/setup.py#L30 for Sphinx 1.8.5) version 1.2 is Python 3 only.